Python/Image To Sepia Effect/sepia image convertor.py

```diff
@@ -0,0 +1,19 @@
+# Importing libraries
+import cv2
+import numpy as np
+
+img = cv2.imread('taj_mahal.jpg')
```

Comment:
Take the input from the user dynamically

Reply:
How should I do that?

Reply:
I tried using Pillow but that's not working :(

Reply:
Well, don't you think, it would be redundant if the user cannot pass it dynamically?

Reply:
Yep, now it worked! Can you please review it again?